db/seeds.prod.ts

```diff
@@ -114,6 +114,21 @@ const seed = async () => {
     update: {},
     create: { name: "Intern" },
   })
+  await db.disciplines.upsert({
```

Comment:
The migrations run before the seeding: https://github.com/wizeline/project-lab/blob/main/scripts/setup.sh#L84

So we probably need to add a couple of inserts in the migration file to add the new Roles. And probably do them as upserts, just in case. Probably something similar to example 2 here: https://www.geeksforgeeks.org/postgresql-upsert/

Reply:
Ooooh! Didn't know that! Thanks for the catch! I just updated the file and added the inserts! (L66).